--- a/Ghidra/Framework/Gui/src/main/java/generic/test/AbstractGuiTest.java
+++ b/Ghidra/Framework/Gui/src/main/java/generic/test/AbstractGuiTest.java
@@ -42,7 +42,6 @@
 import ghidra.util.task.AbstractSwingUpdateManager;
 import ghidra.util.task.SwingUpdateManager;
 import junit.framework.AssertionFailedError;
-import sun.awt.AppContext;
 import utility.function.ExceptionalCallback;
 
 /**
@@ -69,11 +68,6 @@
 	 */
 	public static Set<Window> getAllWindows() {
 		Set<Window> set = new HashSet<>();
-		Frame sharedOwnerFrame = (Frame) AppContext.getAppContext()
-				.get(new StringBuffer("SwingUtilities.sharedOwnerFrame"));
-		if (sharedOwnerFrame != null) {
-			set.addAll(getAllWindows(sharedOwnerFrame));
-		}
 
 		for (Frame frame : Frame.getFrames()) {
 			set.addAll(getAllWindows(frame));
